arches/app/media/js/views/components/geocoders/base-geocoder.js

```diff
@@ -10,6 +10,12 @@ function (ko, mapboxgl, arches) {
      */
     var BaseGeocoderViewModel = function(params) {
         var self = this;
+        let mapboxgl;
+
+        require(['mapbox-gl'], (mapbox) => {
```

Comment:
this is probably safe from race conditions given the way the module is used internal to this file (it is only called when self.map is available, which means this module has been loaded), so I think in this case it's fine.